Update plugin to comply with Yazi 4.0 standards

Pahasara opened the pull request:
Resolves #2

mgumz commented on README.md:
````diff
@@ -4,23 +4,14 @@ Plugin for `Yazi` to preview all supported files via `Bat`.
 ```sh
 $> mkdir -p ~/.config/yazi/plugins
-$> git clone https://github.com/mgumz/yazi-plugin-bat.git ~/.config/yazi/plugins/bat.yazi
+$> git clone https://github.com/Pahasara/yazi-plugin-bat.git ~/.config/yazi/plugins/bat.yazi
````
why would i merge this change of the url?

Pahasara replied:
Sorry about that! I unintentionally left the URL pointing to my fork. I'll fix it right away and update the PR.

mgumz commented on README.md:
```diff
@@ -4,17 +4,8 @@ Plugin for `Yazi` to preview all supported files via `Bat`.
 ## Preview
 
-Showing .csv files
+![preview](https://github.com/user-attachments/assets/67fe6e52-95c6-4768-a378-732e9c11cb84)
```
The preview image is hosted outside of the repo. Please put the preview image into the repo

mgumz commented on init.lua:
```diff
@@ -1,53 +1,54 @@
-        :stdout(Command.PIPED)
-        :stderr(Command.PIPED)
-        :spawn()
+function M:peek(job)
```
The change of the indentation level makes it hard t find the real difference between the versions.

Is the main change the "self" to "job" transition?

Pahasara replied:
Yeah, the main changes are the transition from self to job and replacing ui.Paragraph with ui.Text. The indentation issue has been fixed.

Pahasara committed:
Update README

Pahasara committed:
Update plugin to comply with Yazi 4.0 standards

- Moved task-related information from self to the job parameter
- Replaced ui.Paragraph with ui.Text

Pahasara committed:
Fixes previewer

mgumz commented:
@Pahasara Thank you very much!
